telegram/_files/inputfile.py

```diff
@@ -35,7 +34,11 @@ class InputFile:
     """This object represents a Telegram InputFile.
 
     .. versionchanged:: 20.0
-        The former attribute ``attach`` was renamed to :attr:`attach_name`.
+
+        * The former attribute ``attach`` was renamed to :attr:`attach_name`.
+        * Method ``is_image`` was removed. If you pass :obj:`bytes` to :paramref:`obj` and would
+          like to have the mime type automatically guessed, please pass :paramref:`filename`
```

Comment:
isn't the `filename` already guessed from [earlier](https://github.com/python-telegram-bot/python-telegram-bot/blob/64de8966adc69da230d5206f6d14b61c499373e4/telegram/_files/inputfile.py#L86)?

Reply:
only if you pass a file-like object, i.e. `open('file', 'rb')` instead of plain bytes (`Path(…).read_bytes()`)